deeppavlov/requirements/spelling.txt

```diff
@@ -1,4 +1,3 @@
 lxml==4.4.2
-python-Levenshtein==0.12.0
```

Comment:
Why remove it?

Reply:
python-Levenshtein is installed to speedup fuzzywuzzy and not required for rapidfuzz anymore. Beside this it is GPLv2 Licensed aswell

Reply:
Looks like it's not used anywhere and definitely not in any of the current spelling correction algorithms, so it should be removed from here, thank you :slightly_smiling_face:

Reply:
it is used by fuzzywuzzy to speed up the ratio calculation.